capsicum-test.h

```diff
@@ -249,6 +249,7 @@ void TestSkipped(const char *testcase, const char *test, const std::string& reas
     const ::testing::TestInfo* const info = ::testing::UnitTest::GetInstance()->current_test_info(); \
     std::cerr << "Skipping " << info->test_case_name() << "::" << info->name() << " because: " << reason << std::endl; \
     TestSkipped(info->test_case_name(), info->name(), reason);          \
+    GTEST_SKIP(); \
```

Comment:
`GTEST_SKIP` is not available in gtest-1.8.1, so this doesn't build using the version of Google Test that's in the repository.

So to use `GTEST_SKIP` we'd need to update to [`gtest-1.10.0`](https://github.com/google/googletest/releases/tag/release-1.10.0), which might be more effort than is worthwhile.

(However: I've not used `GTEST_SKIP` &ndash; if it were available, would that mean that the hand-rolled test-skipping stuff here could just be removed altogether, and the `return`s removed from the places that use `TEST_SKIPPED`?  That might make it worthwhile, as long as upgrading Google Test doesn't cause any other problems.)

Reply:
Playing with a full move over at #44

Reply:
If the `TEST_SKIPPED` was hand-rolled before `GTEST_SKIP` has been introduced. I would suggest migration to it, but please note that it doesn't show the reason output as `TEST_SKIPPED` (or I haven't known how to do so).  Here are the last few lines of the current output:

```
[----------] Global test environment tear-down
[==========] 95 tests from 32 test cases ran. (35298 ms total)
[  PASSED  ] 92 tests.
[  SKIPPED ] 3 tests, listed below:
[  SKIPPED ] Capability.SyscallAt
[  SKIPPED ] Capability.NoBypassDAC
[  SKIPPED ] PipePdfork.WildcardWait
```

Reply:
In #44 I took the simple path of just changing the test names to give a suffix that hints as to why they might be skipped, e.g.
```
[  SKIPPED ] Capability.SyscallAtIfRoot
```